### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by elmar8287**
- Create a method to decode a Morse code character, take a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

- Create a method to decode an entire word in Morse code, take a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

- Create a method to decode the entire message in Morse code, take a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g. decode("-- -.--   -. .- -- .") returns "MY NAME").

- Decode the following message:

`.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-. / .-. ..- -... .. . ...`

### CODE REVIEW

File: decoder.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+end
+
+def decode_message(morse_text)
+  message = []
+  morse_text.split('   ').each do |word|
+    message.push(decode_word(word))
+  end
+  message.join(' ')
+end
+
```
Comment (SajjadAhmad14) [inline decoder.rb:34]: Please add a `decode` method which uses `\#decode_word`. `p decode("-- -.-- -. .- -- .")` should return `"MY WORD" and p decode(".- -... --- -..- ..-. ..- .-.. .-.. --- ..-. .-. ..- -... .. . ...")` should return `"A BOX FULL OF RUBIES"`

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Elmar and Kossi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git Flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: decoder.rb (hunk 1 above)
Comment (elmar8287) [reply decoder.rb:34]: Dear @SajjadAhmad14 👍 

Thanks for the review. It is fixed.

Regards

Comment (henatan99) [reply decoder.rb:34]: Nice work! This issue has been addressed well! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @elmar8287 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Many thanks for the review! 👍

File: decoder.rb (hunk 1 above)
Comment (elmar8287) [reply decoder.rb:34]: 👍 👍 👍

**Pull request merged**